RetailCoder.VBE/UI/FindSymbol/FindSymbolControl.xaml.cs

```diff
@@ -43,5 +44,10 @@ private void UIElement_OnPreviewKeyDown(object sender, KeyEventArgs e)
                 e.Handled = true;
             }
         }
+
+        private void FindSymbolControl_Loaded(object sender, System.Windows.RoutedEventArgs e)
+        {
+            this.searchComboBox.Focus();
```

Comment:
`this` qualifier is redundant. Please remove; RD code base doesn't use `this` as a qualifier.